src/hid/midi_parser.h

```diff
@@ -47,9 +48,15 @@ class MidiParser
         ParserSysEx,
     };
 
-    ParserState     pstate_;
-    MidiEvent       incoming_message_;
-    MidiMessageType running_status_;
+    ParserState                             pstate_;
+    MidiEvent                               incoming_message_;
+    MidiMessageType                         running_status_;
+    RingBuffer<uint8_t, SYSEX_BUF_MAX_SIZE> sysex_buf_;
```

Comment:
If you wanted even further configurability, the max size here could be templated (and even potentially set to 0 to ignore sysex on a given MIDI handler/parser).